cmd/configschema/overrides/receiver-filelog.yaml

```diff
@@ -0,0 +1,31 @@
+type: '*filelogreceiver.FileLogConfig'
```

Comment:
what is this - is it a go type? Should all components be in this folder eventually?

Reply:
A README in this folder explaining what this does would help.

Reply:
This is just for overrides to what configschema would normally provide for components with vague types on their configs. I've added a README in this directory to explain.